clang/runtime/dpct-rt/include/dpct/group_utils.hpp

```diff
@@ -517,16 +516,118 @@ class workgroup_load {
   __dpct_inline__ void load(const Item &item, InputIteratorT block_itr,
                             InputT (&items)[ITEMS_PER_WORK_ITEM]) {
 
-    if constexpr (ALGORITHM == BLOCK_LOAD_DIRECT) {
+    if constexpr (ALGORITHM == load_algorithm::BLOCK_LOAD_DIRECT) {
       load_blocked<ITEMS_PER_WORK_ITEM>(item, block_itr, items);
-    } else if constexpr (ALGORITHM == BLOCK_LOAD_STRIPED) {
+    } else if constexpr (ALGORITHM == load_algorithm::BLOCK_LOAD_STRIPED) {
       load_striped<ITEMS_PER_WORK_ITEM>(item, block_itr, items);
     }
   }
 
 private:
   uint8_t *_local_memory;
 };
+
+/// Store blocked/warped or striped work items into linear segment of items.
+/// Helper for Block Store
+enum class store_algorithm {
+
+  BLOCK_STORE_DIRECT,
+  BLOCK_STORE_STRIPED,
+
+};
+
+/// Stores a blocked arrangement of work items linear segment of items.
```

Comment:
I'd like to have a more detail comments like https://github.com/oneapi-src/SYCLomatic/blob/b2e55881f40b1fb1e422b095e68f28b5d148ae1a/clang/runtime/dpct-rt/include/dpct/sparse_utils.hpp#L278-L306